src/main/java/it/gov/pagopa/pu/classification/dto/ClassificationDTO.java

```diff
@@ -0,0 +1,47 @@
+package it.gov.pagopa.pu.classification.dto;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import java.time.OffsetDateTime;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class ClassificationDTO {
+
+  private String iuf;
+  private String iud;
+  private String iuv;
+  private String iur;
+  @NotNull
+  private String label;
+
+  private OffsetDateTime lastClassificationDateFrom;
+  private OffsetDateTime lastClassificationDateTo;
```

Comment:
Use OffsetDateTimeIntervalFilter instead of this approach to configure from and to